deploy/crd/cluster.open-cluster-management.io_clustercurators.yaml

```diff
@@ -1,9 +1,10 @@
+
```

Comment:
```suggestion
```

lets get rid of this extra space just in case.

Reply:
Sounds good. Fixed.